frontend/src/metabase/core/components/Link/Link.styled.tsx

```diff
@@ -18,3 +19,20 @@ export const LinkRoot = styled(Link, {
 
   ${focusOutlineStyle("brand")};
 `;
+
+export const variants = {
```

Comment:
This is fine for now but can become problematic if we need to add additional props to links. Wdyt about the following:

```
const LinkRoot = styled(Link)<LinkRootProps>`
  ...

  ${props => props.variant === "brand" && css`
    color: ${metabaseColor("brand")};
    &:hover {
      text-decoration: underline;
    }
  `}

  ${props => props.variant === "brandBold" && css`
     ...
  `}
`;
```

or

```
const LinkRoot = styled(Link)<LinkRootProps>`
  ...

  ${props => props.variant !== "default" && css`
    color: ${metabaseColor("brand")};
    &:hover {
      text-decoration: underline;
    }
  `}

  ${props => props.variant === "brandBold" && css`
     fontWeight: bold;
  `}
`;
```